libs/templates/src/lib/trade/app-download/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Heading, Text } from '@deriv/quill-design';
+import { Card } from '@deriv-com/components';
+import { data } from './data';
+
+const AppDownload = () => {
+  const { content, heading, description, cta, platforms } = data;
+
+  return (
+    <Card.ContentRight
+      color="gray"
+      align="start"
+      size="lg"
+      content={content}
+      className="items-end lg:gap-gap-lg"
+      header="title here"
+      // description="description here"
+    >
+      <div className="flex flex-col gap-gap-xl lg:gap-gap-2xl">
```

Comment:
semantic will handle this.
```suggestion
      <div className="flex flex-col gap-gap-2xl">
```

Reply:
done changes